pkg/hoist/hoist_launchable.go

```diff
@@ -98,7 +112,7 @@ func (hl *Launchable) enable() (string, error) {
 
 	// providing an enable script is optional, ignore those errors
 	if err != nil && !os.IsNotExist(err) {
-		return output, util.Errorf("Could not enable %s: %s", hl.Id, err)
```

Comment:
I like stack traces for our errors because it makes it easy for https://getsentry.com/welcome/ to de duplicate them. Otherwise sure, it's pretty obvious where it's originating from.